deployer/src/deployer/upload.py

```diff
@@ -498,6 +498,16 @@ def delete(self, keys, on_task_complete=None, dry_run=False):
         return timer
 
 
+def parse_archived_txt_file(file: Path):
+    file_paths = []
+    with open(file) as f:
+        for line in f:
+            line = line.strip()
+            if line and not line.startswith("#"):
+                file_paths.append(line)
+    return file_paths
```

Comment:
I'm thinking we should make this a generator (since the file has over 16K entries):

```suggestion
    with open(file) as f:
        for line in f:
            line = line.strip()
            if line and not line.startswith("#"):
                yield line
```

and then also modify how we use it (see additional comments below).

Reply:
I disagree. There is no point in avoiding this being loaded into RAM. Eventually, we need and want a complete set so it's speedy to look things up so it'll need to be all fully loaded eventually. 
The only answer to "speedy to look things up" is if something is a dict or a set. 

What you're propose it akin to this:
```python
def generator():
    with open(filename) as f:
      for line in f:
          yield line.strip()
things = set(x for x in generator)
```
That is actually identical to:
```python
def list():
    stuff = []
    with open(filename) as f:
      for line in f:
          stuff.append(line.strip())
things = set(list())
```

However, I digress. My code produces a 16k list **and** a 16k set. 
I'll take a look.